### ISSUE TITLE
Are the 4.7 patches ready to go?

### ISSUE THREAD

**Issue by morbus** [freeform/civicrm-drupal-pantheon#2]
G'day. Thinking of using your CiviCRM install as a basis for an (existing Drupal site that will have CiviCRM added to it) on Pantheon. It is unclear to me if the 4.7.x. branch, however, is "ready to go" patch-wise.

**Comment by herbdool** [comment]
It's not ready to go. We've got a couple clients using CiviCRM on Pantheon but neither has moved to 4.7 yet. I've got no timeline yet of getting it working. I'm not sure how much work it'll be yet. Any chance you've got a few spare moments to help?

**Comment by morbus** [comment]
We're currently in the process of evaluating the "costs" of adding CiviCRM to an existing site on Pantheon vs. moving to another web host entirely. If we do decide to go the Pantheon route, with 4.7, aye, we'll definitely contribute back know-how.

**Comment by herbdool** [comment]
So your question got me going for an hour. I've got the patches which I think are required updated. I've also added the optional patch for storing Smarty templates in Redis, if you've got Redis enabled (I've found it conventional but works well enough). (You'd have to re-compile with drush make to include it).

I haven't tested it at all.

There was one patch that I didn't upgrade and might not be needed, since the way that config variables are stored has changed quite a bit and should hopefully be handled better by CiviCRM in 4.7 when running on Pantheon.
